runtime/common/BaseRemoteRESTQPU.h

```diff
@@ -402,10 +417,26 @@ class BaseRemoteRESTQPU : public cudaq::QPU {
         throw std::runtime_error("Remote rest platform Quake lowering failed.");
     };
 
-    if (updatedArgs) {
-      cudaq::info("Run Quake Synth.\n");
+    if (!rawArgs.empty() || updatedArgs) {
       mlir::PassManager pm(&context);
-      pm.addPass(cudaq::opt::createQuakeSynthesizer(kernelName, updatedArgs));
+      if (!rawArgs.empty()) {
```

Comment:
Should there be a similar change in py_alt_launch_kernel.cpp (in `synthesizeKernel`)?

Reply:
Yes. This PR gets one of the many flows up to the point it can be tested end-to-end. But we need to tackle all of the different flows. (I think that work can be done in parallel—finally.)

Reply:
sounds good (and looks good to me)